tests/run/partial_circular_import.srctree

```diff
@@ -0,0 +1,23 @@
+PYTHON setup.py build_ext --inplace
```

Comment:
```suggestion
PYTHON -c "import pkg.A"
PYTHON setup.py build_ext --inplace
```

For these tests where we're aiming to match the behaviour to Python we often try to run the test in Python too.